packages/core/admin/admin/src/utils/createRoute.js

```diff
@@ -12,7 +12,7 @@ const LazyCompo = ({ loadComponent }) => {
       try {
         const loadedCompo = await loadComponent();
 
-        setCompo(() => loadedCompo.default);
+        setCompo(() => loadedCompo?.default ?? loadedCompo);
```

Comment:
This allows us to use named exports as the plugin `Component` as I have here with email. wdyt of this? @joshuaellis @gu-stav

Reply:
I think the last time we tried to change this, we had a lot of issues... @gu-stav knows more than I however.

Reply:
Gosh this util again. Mmh - I'm not sure I like it, because it would mean a default export is always preferred, if one is set, which creates some ambiguity when using it. I think it would be fine for me, if you add an extensive code-comment about what is going on here?

Reply:
👍🏻 is this okay [`a5a635c` (#18127)](https://github.com/strapi/strapi/pull/18127/commits/a5a635c14d1e44276abd7eb4098217af9aea0a8c)

Reply:
im down.